redis-pq-demo/docker-compose.redis.yml

```diff
@@ -0,0 +1,112 @@
+services:
+  redis-cluster:
+    image: redis:7
+    container_name: redis-cluster
+    command: redis-cli --cluster create 172.28.0.101:6377 172.28.0.102:6378 172.28.0.103:6379 172.28.0.104:6380 172.28.0.105:6381 172.28.0.106:6382 --cluster-replicas 1 --cluster-yes
+    environment:
+      - REDISCLI_AUTH=${REDIS_PASSWORD}
+    networks:
+        acapy_default:
+          ipv4_address: 172.28.0.107
+    depends_on:
+      - redis-node-1
```

Comment:
Is this allot a nodes for a local / docker instance?

Reply:
I believe a redi cluster requires a minimum of 6 nodes.